WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -95,4 +128,35 @@
   });
 
   $('#display-<%= @display %>').button('toggle');
+
+  // From http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+  function getParameterByName(name, url) {
+    if (!url) url = window.location.href;
+    name = name.replace(/[\[\]]/g, "\\$&");
+    var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+        results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, " "));
+  }
+
+  function display_selectors() {
+    $display_class_from_url = getParameterByName('display');
```

Comment:
Missing the `var` keyword here... also, our convention is to only prefix variable names with `$` if they're a jquery object, and that does not appear to be the case here.

Lastly, our javascript convention is to use `camelCase`, not `snake_case`